packages/contact-center/task/src/CallControl/call-control.presentational.tsx

```diff
@@ -45,50 +46,117 @@ function CallControlPresentational(props: CallControlPresentationalProps) {
+        {!wrapupRequired && (
+          <div className="button-group">
+            {buttons.map((button, index) => (
+              <TooltipNext
+                key={index}
+                color="primary"
+                delay={[0, 0]}
+                placement="bottom-start"
+                triggerComponent={
+                  <ButtonCircle className={button.className} onPress={button.onClick}>
+                    <Icon className={button.className + '-icon'} name={button.icon} />
+                  </ButtonCircle>
+                }
+                type="description"
+                variant="small"
+                className="tooltip"
+              >
+                <p>{button.tooltip}</p>
+              </TooltipNext>
+            ))}
+          </div>
+        )}
+        {wrapupRequired && (
+          <div className="wrapup-group">
+            <PopoverNext
+              color="primary"
+              delay={[0, 0]}
+              placement="bottom-start"
+              showArrow
+              trigger="click"
+              triggerComponent={
+                <ButtonPill onPress={handleWrapupCall} className="wrapup-button">
+                  Wrap up
+                  <Icon name="arrow-down-bold" />
+                </ButtonPill>
+              }
+              variant="medium"
+              interactive
+              offsetDistance={2}
+              className="wrapup-popover"
+            >
+              <Text className="wrapup-header" tagName={'small'} type="subheader-primary">
+                Wrap-up Interaction
+              </Text>
+              <div className="wrapup-select-wrapper">
+                <SelectNext
+                  aria-label="wrapup-reason"
+                  className="wrapup-select"
+                  onSelectionChange={(key) => {
+                    const selectedItem = wrapupCodes?.find((code) => code.id === key);
+                    handleWrapupChange(selectedItem.name, selectedItem.id);
+                  }}
+                  items={wrapupCodes}
+                  showBorder={false}
+                  placeholder="Select"
+                  label="Wrap-up reason"
+                >
+                  {(item) => (
+                    <Item key={item.id} textValue={item.name}>
+                      <Text className="wrapup-name" tagName={'small'}>
+                        {item.name}
+                      </Text>
+                    </Item>
+                  )}
+                </SelectNext>
+                <Icon className="wrapup-select-arrow-icon" name="arrow-down-bold" title="" />
               </div>
-            </fieldset>
-          </section>
-        </div>
-      )}
+              <ButtonPill
+                className="submit-wrapup-button"
+                onPress={() => wrapupCall(selectedWrapupReason, selectedWrapupId)}
+                disabled={selectedWrapupId && selectedWrapupReason ? false : true}
+              >
+                submit & Wrapup
```

Comment:
```suggestion
                Submit & Wrap up
```

Also ensure that this comes from the constant file

Reply:
Updated